src/transformers/utils/generic.py

```diff
@@ -289,3 +290,23 @@ def __enter__(self):
 
     def __exit__(self, *args, **kwargs):
         self.stack.__exit__(*args, **kwargs)
+
+
+def find_labels(model_class):
+    """
+    Find the labels used by a given model.
+
+    Args:
+        model_class (`type`): The class of the model.
+    """
+    model_name = model_class.__name__
+    if model_name.startswith("TF"):
+        signature = inspect.signature(model_class.call)
+    elif model_name.startswith("Flax"):
+        signature = inspect.signature(model_class.__call__)
+    else:
+        signature = inspect.signature(model_class.forward)
+    if "QuestionAnswering" in model_name:
+        return [p for p in signature.parameters if "label" in p or p == "start_positions" or p == "end_positions"]
```

Comment:
```suggestion
        return [p for p in signature.parameters if "label" in p or p in ("start_positions", "end_positions")]
```
This is the nittiest nit ever but it's a little cleaner!

Reply:
Indeed!